src/Framework/AspNetCoreAnalyzers/src/Analyzers/DiagnosticDescriptors.cs

```diff
@@ -106,4 +106,13 @@ internal static class DiagnosticDescriptors
         DiagnosticSeverity.Warning,
         isEnabledByDefault: true,
         helpLinkUri: "https://aka.ms/aspnet/analyzers");
+
+    internal static readonly DiagnosticDescriptor UseTopLevelRouteRegistrationsInsteadOfUseEndpoints = new(
+        "ASP0014",
+        "Suggest using app.MapGet over of using app.UseEndpoints",
```

Comment:
This message should be more generic to cover other Map methods, e.g. MapPost, MapPut, etc.